packages/venia-concept/src/components/Checkout/form.js

```diff
@@ -238,6 +230,11 @@ class Form extends Component {
         return <div className={classes.root}>{children}</div>;
     }
 
+    cancelHandler = () => {
+        const { closeCheckoutForm } = this.props;
+        closeCheckoutForm();
```

Comment:
Please add `closeCheckoutForm` to this component's `propTypes`. 

Since we're calling it directly here, please also make it `required`.